lib/wanda/executions/evaluation.ex

```diff
@@ -206,49 +208,109 @@ defmodule Wanda.Executions.Evaluation do
     end
   end
 
+  defp normalize_return_value(:expect_enum, "passing"), do: :passing
+  defp normalize_return_value(:expect_enum, "warning"), do: :warning
+  defp normalize_return_value(:expect_enum, _), do: :critical
+  defp normalize_return_value(_, return_value), do: return_value
+
   defp maybe_add_failure_message(
          %ExpectationEvaluation{type: :expect, return_value: false} = expectation_evaluation,
          nil,
+         _warning_message,
          _evaluation_scope,
          _
-       ) do
-    %ExpectationEvaluation{expectation_evaluation | failure_message: @default_failure_message}
-  end
+       ),
+       do: %ExpectationEvaluation{
+         expectation_evaluation
+         | failure_message: @default_failure_message
+       }
 
   defp maybe_add_failure_message(
          %ExpectationEvaluation{type: :expect, return_value: false} = expectation_evaluation,
          failure_message,
+         _warning_message,
          evaluation_scope,
          engine
-       ) do
-    message =
-      case EvaluationEngine.eval(engine, "`#{failure_message}`", evaluation_scope) do
-        {:ok, interpolated_failure_message} -> interpolated_failure_message
-        _ -> @default_failure_message
-      end
+       ),
+       do: %ExpectationEvaluation{
+         expectation_evaluation
+         | failure_message: interpolate_message(failure_message, evaluation_scope, engine)
+       }
 
-    %ExpectationEvaluation{expectation_evaluation | failure_message: message}
-  end
+  defp maybe_add_failure_message(
+         %ExpectationEvaluation{type: :expect_enum, return_value: :passing} =
+           expectation_evaluation,
+         _,
+         _,
+         _evaluation_scope,
+         _
+       ),
+       do: expectation_evaluation
+
+  defp maybe_add_failure_message(
```

Comment:
Could we have this at the end of `maybe_add_failure_message` expect_enum clauses?, so we have expect enum with passing, critical and warning, and then this "default"

Reply:
we cannot really do as the next 2 functions would override this all the times.
We need to check for the `nil` values.
But let me have a look on that. To see if I can improve it somehow (but i'm not sure I can)